example_site/src/components/ConfigurationControls.js

```diff
@@ -134,306 +150,348 @@ export default function ConfigurationControls({ isOpen }) {
 
   const formClasses = classnames({
     "configuration-controls-form": true,
-    panel: true,
-    open: isOpen,
-    closed: !isOpen,
+    'modal-fade': true,
+     panel: true,
+     open: isOpen,
+     closed: !isOpen,
+  });
+
+  const titlePanel = classnames({
+    "title-description": true,
+     panel: true,
+     open: isOpen,
+     closed: !isOpen,
   });
 
   return (
-    <form onSubmit={handleSubmit(onSubmit)} className={formClasses}>
-      <div>
-        <h2>Configuration</h2>
+    <form onSubmit={handleSubmit(onSubmit)} className={titlePanel}>
+      <div className='center-title'>
+        <div className='Header'>
+          <Header 
+            title="Welcome to the Waymark SDK"
+            subtitle="This demo site is intended to give you a sense of 
+            what you can do with Waymark's SDK.
+            The possibilities are limitless, but hopefully 
+            this gets your gears turning about how the
+            Waymark SDK could work for you."
+          />
+        </div>
+
+        <h4>To get started, choose an example partner</h4>
 
-        <label title="Environment" className="form-label" htmlFor="environment">
-          Environment Connection
-        </label>
         <select
-          name="environment"
+          name="partner"
           ref={register({ required: true })}
-          defaultValue="demo"
+          value={partner}
+          onChange={(e) => setPartner(e.target.value)}
         >
-          <option value="demo">Demo</option>
-          <option value="prod">Production</option>
-          <option value="local">Local</option>
+          <option selected value="default">Default Partner</option>
+          <option value="spectrum">Spectrum Reach</option>
+          <option value="custom">Custom</option>
```

Comment:
We've been bad developers (me ryan and scott, not you)! We should define these partners, and the environments from the environment dropdown as constants like we do for the configurations.